tests/compiler/closure-passing-functions.ts

```diff
@@ -0,0 +1,12 @@
+function add(x: i32, y: i32): i32 { 
+  return x + y; 
+}
+function apply_to_pair(x: i32, y: i32, fn: (x:i32, y:i32) => i32): i32 { 
+  return fn(x,y);
+}
+
+let addResult = add(1, 1);
+let applyResult = apply_to_pair(1, 2, add);
+
+assert(addResult == 2);
+assert(applyResult == 3);
```

Comment:
I don't see the closure in this test 🤔  what is the purpose of this test?

Reply:
Ah! Good catch, yeah we added this test because of our current implementation in #1308 had a problem with this test. But, this was before we fixed the CI, so this test probably isn't needed. I'll remove it :smile: